iac/tf-multienv-cicd-anthos-autopilot/shell-scripts/01-patch-acm-config.sh

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Google LLC
```

Comment:
I'm not super sure about this script -- I think I'd prefer if we'd hardcode (or default a variable) to this current GCP project / region and not have this script at all. I'd rather `git` commands (besides `git clone`) not be involved.

Reply:
Perhaps instead of a script, it could be part of the README, where in the prereq you state that if you're deploying this TF outside the context of BoA maintainers, you should:
- fork the repo
- substitute some stuff like the GCP project name and commit those in your now forked repo (perhaps that bit can be a quick script, but it should not be part of the TF family of script) -- though I'd personally lean towards simplifying and not scripting it at all... Just stating it in the README
- THEN you can run the TF scripts

Reply:
I agree. These scripts were mostly created to support me in testing reproducibility of the setup with little effort. For this PR I have therefore removed all bash scripts and instead added step-by-step instructions for the initial (one-time) setup to the README.md.